<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HashTableUI</class>
 <widget class="QMainWindow" name="HashTableUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HashTableUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="moduleSpinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000000000</number>
    </property>
    <property name="value">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="moduleButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statisticButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>statistic</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>add</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>remove</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>find</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="valueLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="executeButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Module</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
